backend/molgenis-emx2-semantics/src/test/java/org/molgenis/emx2/semantics/beaconv2/Beaconv2_EntryTypesTest.java

```diff
@@ -0,0 +1,87 @@
+package org.molgenis.emx2.semantics.beaconv2;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.junit.Test;
+import org.molgenis.emx2.beaconv2.endpoints.EntryTypes;
+import org.molgenis.emx2.json.JsonUtil;
+import spark.Request;
+
+public class Beaconv2_EntryTypesTest {
+
+  @Test
+  public void testEntryTypes() throws Exception {
+    Request request = mock(Request.class);
+    when(request.url()).thenReturn("http://localhost:8080/myEmx2Schema/api/beacon/map");
+    EntryTypes et = new EntryTypes();
+    String json = JsonUtil.getWriter().writeValueAsString(et);
+
+    // header
+    assertTrue(
+        json.contains(
+            "  \"meta\" : {\n"
```

Comment:
For the future if you like it Java supports multiline strings. E.g. see here https://stackoverflow.com/questions/878573/does-java-have-support-for-multiline-strings/50155171#50155171

Just FYI, no action needed

Reply:
fixed, now uses text blocks